### ISSUE TITLE
Deleting then uploading file of same name and type reuploads first file

### ISSUE THREAD

**Issue by testerpester2** [filegarden/website#1]
https://github.com/user-attachments/assets/a629e34f-b380-4b8a-a895-010bd78664a6

**Comment by GrantGryczan** [comment]
Good catch! But unfortunately this isn't the right place to report this bug; this repository is for the (currently WIP) rewrite of File Garden's website, not the version that's currently live. This bug might not exist anymore once the new version is released.
